process/profile.go

```diff
@@ -72,7 +72,9 @@ func (p *Process) GetProfile(ctx context.Context) (changed bool, err error) {
 			case "/lib/systemd/systemd-resolved",
 				"/usr/lib/systemd/systemd-resolved",
 				"/lib64/systemd/systemd-resolved",
-				"/usr/lib64/systemd/systemd-resolved":
+				"/usr/lib64/systemd/systemd-resolved",
+				"/usr/sbin/nscd",
+				"/usr/local/sbin/nscd":
```

Comment:
I'm not sure we should consider binaries in `/usr/local` here. No package manager will install something there users that install software by compiling and (likely) `make install`ing them can also create symlinks for that. We might open up some security issues if we consider `/usr/local` as well.